docs/data/material/components/slider/MusicPlayerSlider.tsx

```diff
@@ -159,40 +159,57 @@ export default function MusicPlayerSlider() {
           <TinyText>-{formatDuration(duration - position)}</TinyText>
         </Box>
         <Box
-          sx={{
+          sx={(theme) => ({
             display: 'flex',
             alignItems: 'center',
             justifyContent: 'center',
             mt: -1,
-          }}
+            '& svg': {
+              color: '#000',
+              ...theme.applyStyles('dark', {
+                color: '#fff',
+              }),
+            },
+          })}
         >
           <IconButton aria-label="previous song">
-            <FastRewindRounded fontSize="large" htmlColor={mainIconColor} />
+            <FastRewindRounded fontSize="large" />
           </IconButton>
           <IconButton
             aria-label={paused ? 'play' : 'pause'}
             onClick={() => setPaused(!paused)}
           >
             {paused ? (
-              <PlayArrowRounded
-                sx={{ fontSize: '3rem' }}
-                htmlColor={mainIconColor}
-              />
+              <PlayArrowRounded sx={{ fontSize: '3rem' }} />
             ) : (
-              <PauseRounded sx={{ fontSize: '3rem' }} htmlColor={mainIconColor} />
+              <PauseRounded sx={{ fontSize: '3rem' }} />
             )}
           </IconButton>
           <IconButton aria-label="next song">
-            <FastForwardRounded fontSize="large" htmlColor={mainIconColor} />
+            <FastForwardRounded fontSize="large" />
           </IconButton>
         </Box>
-        <Stack spacing={2} direction="row" sx={{ mb: 1, px: 1 }} alignItems="center">
-          <VolumeDownRounded htmlColor={lightIconColor} />
+        <Stack
+          spacing={2}
+          direction="row"
+          sx={(theme) => ({
+            mb: 1,
+            px: 1,
+            '& > svg': {
+              color: 'rgba(0,0,0,0.4)',
+              ...theme.applyStyles('dark', {
+                color: 'rgba(255,255,255,0.4)',
+              }),
+            },
+          })}
+          alignItems="center"
+        >
+          <VolumeDownRounded />
           <Slider
             aria-label="Volume"
             defaultValue={30}
-            sx={{
-              color: theme.palette.mode === 'dark' ? '#fff' : 'rgba(0,0,0,0.87)',
+            sx={(t) => ({
```

Comment:
Why is the name `t` here?

Reply:
There is `const theme = useTheme()` above this element, I can't use `theme` here due to lint error.

Reply:
Ah got ya, alright!